prov/efa/src/rxr/rxr_cq.c

```diff
@@ -1079,8 +1079,8 @@ static void rxr_cq_handle_rts(struct rxr_ep *ep,
 		} else if (OFI_UNLIKELY(ret == -FI_EALREADY)) {
 			FI_WARN(&rxr_prov, FI_LOG_EP_CTRL,
 				"Duplicate RTS packet msg_id: %" PRIu32
-				" next_msg_id: %" PRIu32 "\n",
-			       rts_hdr->msg_id, peer->next_msg_id);
+				" next_msg_id: %" PRIu64 "\n",
```

Comment:
Can you change the text in the print too (`next_msg_id`->`exp_msg_id`)?

Reply:
done